### ISSUE TITLE
conflicting namespace prefix in geodcat-ap-SHACL.ttl

### ISSUE THREAD

**Issue by NielsHoffmann** [SEMICeu/GeoDCAT-AP#145]
This is an issue from the GeoDCAT-AP 3.0.0 pilot.

In the geodcat-ap-SHACL.ttl file the following is declared:
`@prefix dc: <http://purl.org/dc/terms/> .`

typically de `dc:` prefix is reserved for `http://purl.org/dc/elements/1.1/`
while `dcterms:`  is the most commonly used prefix for 'http://purl.org/dc/terms/`

The use of a conflicting prefix causes issues with the use of python rdflib and pyshacl for validation.
While I would prefer dcterms as prefix, I see the xslt is declaring dct: which is used inside the uri's of the nodeShapes as well. 
Using dct would solve the conflict with rdflib/pyshacl as well.

**Comment by jakubklimek** [comment]
I agree that there is currently a common practice misalignment (`dc` vs `dct` or `dcterms`).

At the same time, this seems to be an issue with the used tooling, because even though the prefix `dc` is not common here, syntactically and semantically, it is correct RDF, so it should not cause issues.

**Comment by jakubklimek** [comment]
The proposed solution is to manually change the prefix in the SHACL file upon publication and investigate, if this can be changed in the SEMIC toolchain. The same issue is present in other specifications as well.
